firebase-app-distribution/src/test/java/com/google/firebase/appdistribution/UpdateApkClientTest.java

```diff
@@ -90,13 +88,16 @@ public void setup() {
 
   @Test
   public void updateApk_whenDownloadFails_setsNetworkError() throws Exception {
+    List<UpdateProgress> progressEvents = new ArrayList<>();
     UpdateTaskImpl updateTask = new UpdateTaskImpl();
+    updateTask.addOnProgressListener(progressEvents::add);
```

Comment:
Should this test assert that the progressListener gets called with download failure?

Reply:
forgot to remove that, it shouldn't since this failure happens before the download is set to pending